Ports/halflife/package.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port="half-life"
+version="1.0.0"
+useconfigure="true"
+depends=("SDL2")
+workdir="xash3d-fwgs-master"
+files="https://github.com/SerenityPorts/xash3d-fwgs/archive/master.tar.gz xash3d_engine.tar.gz
+https://github.com/SerenityPorts/hlsdk-xash3d/archive/master.tar.gz xash3d_hldll.tar.gz"
+launcher_name="Half-Life"
+launcher_category="Games"
+launcher_command="sh /home/anon/Games/halflife/hl.sh"
+
+# This one is a bit tricky to build, so I'm going a little bit off the script....
+configure() {
+    # Configure the shared object projects (client and game)
+    cd ./hlsdk-xash3d-master
+    ./waf configure -T release
+    cd ../
+
+    # Configure the engine itself...
+    cd ./xash3d-fwgs-master
+    ./waf configure --sdl2=../../SDL2/SDL-main-serenity/ -T release
```

Comment:
Looks like the only issue here is that waf doesn't have any idea that we are cross-compiling and therefore tries to link against the host SDL2 (as indicated by `sdl2-config`). Just manually pointing it to the SDL prefix on the disk fixes that, and it looks a lot more close to what you would expect.

```suggestion
    ./waf configure --sdl2=${SERENITY_INSTALL_ROOT}/usr/local -T release
```

Reply:
Ah, I think I might've been pointing to `/usr/lib/` not `/usr/local`!